media/css/firefox/home/protocol.scss

```diff
@@ -0,0 +1,340 @@
+@import '../../../protocol/css/components/feature-card';
+@import '../../../protocol/css/components/hero';
+@import '../../../protocol/css/components/call-out';
+@import '../../../protocol/css/components/modal';
+
+
+//* -------------------------------------------------------------------------- */
+// Page specific variables and over-rides
+
+$fx-body: #42425A;
+$fx-coral: #FF4F5D;
+$fx-orange: #FF7139;
+$fx-pink: #FF2889;
+$fx-purple: #20123A;
+
+.mzp-t-firefox {
+    &,
+    h1,
+    h2,
+    h3,
+    h4 {
+        font-family: system-ui, sans-serif;
+    }
+}
+
+main .mzp-l-content {
+    max-width: 400px;
+
+    @media #{$mq-md} {
+        max-width: $content-max;
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Hero
+
+.mzp-c-hero {
+    text-align: center;
+    color: $fx-purple;
+
+    &.mzp-t-firefox::after {
+        display: none;
+    }
+}
+
+.mzp-c-hero-title {
+    @include text-display-sm;
+    font-weight: bold;
+    padding-top: 0;
+
+    &:before {
+        @include at2x('#{$image-path}/logos/firefox/firefox.png', 48px, 48px);
+        background-repeat: no-repeat;
+        content: '';
+        display: inline-block;
+        height: 48px;
+        margin-top: -3px;
+        padding-right: $spacing-sm;
+        vertical-align: middle;
+        width: 48px;
+    }
+
+    strong {
+        font-weight: normal;
+    }
+}
+
+.mzp-c-hero-desc {
+    @include text-display-xl;
+    font-weight: bold;
+    margin-bottom: $spacing-md;
+    color: $fx-purple;
+
+    p {
+        margin-bottom: $spacing-lg;
+    }
+}
+
+.mzp-c-button-download-container {
+    margin-bottom: 0;
+}
+
+@media #{$mq-md} {
+    .mzp-c-hero.mzp-has-image {
+        align-items: center;
+        background: top left 50vw no-repeat url('/media/img/firefox/home/hero-small.jpg');
+        background-size: contain;
+        display: flex;
+        min-height: 560px;
+        text-align: left;
+
+        @media all #{$mq-high-res} {
+            background: top left 50vw no-repeat url('/media/img/firefox/home/hero-small-high-res.jpg');
+            background-size: contain;
+        }
+    }
+
+    .mzp-l-content {
+        width: 100%;
+    }
+}
+
+@media #{$mq-lg} and #{$mq-tall} {
+    .mzp-c-hero.mzp-has-image {
+        background: top left 50vw no-repeat url('/media/img/firefox/home/hero.jpg');
+        background-size: contain;
+        height: 60vh;
+        max-height: 672px;
+
+        @media all #{$mq-high-res} {
+            background: top left 50vw no-repeat url('/media/img/firefox/home/hero-high-res.jpg');
+            background-size: contain;
+        }
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Feature cards
+
+.mzp-c-card-feature {
+    .mzp-c-card-feature-media-wrapper,
+    .mzp-c-card-feature-content {
+        min-width: 0; //https://stackoverflow.com/questions/43311943/prevent-content-from-expanding-grid-items
```

Comment:
ok to leave the stackoverflow comment here?

Reply:
I wanted people to understand why I'd added the min-width. I added "because of" to the comment. Hopefully this makes it more clear.